apps/react-app/src/pods/manfred-json-input/manfred-json-input.container.tsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
 import { ManfredJsonInput } from './manfred-json-input.component';
+import { switchRoutes } from '@/core';
 
 export const ManfredJsonInputContainer: React.FC = () => {
-  return <ManfredJsonInput />;
+
+  const [data, setData] = useState('');
+  const navigate = useNavigate();
+
+
+  const handleData = (text:string) => {
+    setData(text);
```

Comment:
Remove this, since we are navigating away this won't have any effect, later on in a different case we will store it in a context